recipe/meta.yaml

```diff
@@ -1,8 +1,12 @@
 {% set name = "xarray" %}
 {% set version = "2024.7.0" %}
+{% set min_python = "python >=3.9" %}
+
+# handle undefined PYTHON in `noarch: generic` outputs
+{% if PYTHON is not defined %}{% set PYTHON = "$PYTHON" %}{% endif %}
 
 package:
-  name: {{ name|lower }}
+  name: xarray-suite
```

Comment:
would this mean that we'd get a separate page here: https://anaconda.org/conda-forge/xarray If so, it might be worth figuring out if we can keep this as `xarray`